docs/migration.md

```diff
@@ -0,0 +1,102 @@
+# Migration from Preview 2
+
+If you are customers of [Cloud Foundry on Azure Preview 2](https://azure.microsoft.com/en-us/blog/cloud-foundry-on-azure-preview-2-now-available/), you need to do the following migration.
+
+## Migration
+
+1. Update bosh.yml.
+
+  Replace **BOSH-FOR-AZURE-URL**, **BOSH-FOR-AZURE-SHA1**, **BOSH-AZURE-CPI-URL**, **BOSH-AZURE-CPI-SHA1**, **STEMCELL-FOR-AZURE-URL** and **STEMCELL-FOR-AZURE-SHA1** with the values in the example file [**bosh.yml**](https://github.com/Azure/azure-quickstart-templates/blob/master/bosh-setup/bosh.yml).
+
+  Click [**HERE**](./deploy-bosh-manually.md#configure_bosh_yml) to learn more.
+
+2. You need to upgrade Ruby version.
```

Comment:
Specify the change should be in dev-box. "You need to upgrade Ruby version from 1.9.3 to 2.0 in your dev-box."